### PR TITLE
add python packages

### PR DESCRIPTION
**Pull request by mylesstokowski**
Adds packages required for converting tiffs to zarr format, which we plan to use in the model comparison viewer.

### CODE REVIEW

**Comment by mylesstokowski** [comment]
fyi @sligocki @iskandari

File: dockerst/Dockerfile [hunk 1]
```diff
@@ -28,7 +28,13 @@ RUN apt-get update -y \
     python3-scipy \
     python3-sklearn \
     python3-tqdm \
-    python3-yaml
+    python3-yaml \
+    python3-gdal \
+    python3-rasterio \
+    python3-xarray \
+    python3-rioxarray \
+    python3-dask \
```
Comment (sligocki) [inline dockerst/Dockerfile:36]: Can you keep this alphabetical?

File: dockerst/Dockerfile [hunk 2]
```diff
@@ -39,7 +45,8 @@ RUN apt-get update -y \
 ENV PIP_BREAK_SYSTEM_PACKAGES 1
 RUN python3 -m pip install \
     pyarrow \
-    torch
+    torch \
+    ndpyramid
```
Comment (sligocki) [inline dockerst/Dockerfile:49]: alphabetical here too

**Review by sligocki — APPROVED** [review]

File: dockerst/Dockerfile [hunk 3]
```diff
@@ -28,7 +28,13 @@ RUN apt-get update -y \
     python3-scipy \
     python3-sklearn \
     python3-tqdm \
-    python3-yaml
+    python3-yaml \
+    python3-gdal \
```
[the remark this answers is not in the document]
Comment (mylesstokowski) [reply dockerst/Dockerfile:32]: Good catch, removed it

**Comment by mylesstokowski** [comment]
The action failed again for a different reason, apparently related to disk space:

`ERROR: failed to solve: ResourceExhausted: mount callback failed on /tmp/containerd-mount1958197008: mount callback failed on /tmp/containerd-mount3775375620: failed to write compressed diff: failed to create diff tar stream: failed to copy: write /var/lib/buildkit/runc-overlayfs/content/ingest/b6ea7222c9fe4f168f7df6fa5effc84e187dbb8be100db5cf8ddfee46b58c50c/data: no space left on device`

https://github.com/ebird/docker/actions/runs/11463880943/job/31903978350?pr=12#step:9:15072

**Comment by sligocki** [comment]
> The action failed again for a different reason, apparently related to disk space:
> 
> `ERROR: failed to solve: ResourceExhausted: mount callback failed on /tmp/containerd-mount1958197008: mount callback failed on /tmp/containerd-mount3775375620: failed to write compressed diff: failed to create diff tar stream: failed to copy: write /var/lib/buildkit/runc-overlayfs/content/ingest/b6ea7222c9fe4f168f7df6fa5effc84e187dbb8be100db5cf8ddfee46b58c50c/data: no space left on device`
> 
> https://github.com/ebird/docker/actions/runs/11463880943/job/31903978350?pr=12#step:9:15072

Sigh, I feel like we've seen this before and suspect it is not your fault. I'm guessing we are close to some disk limit and keep hitting it with small changes?

**Comment by mylesstokowski** [comment]
> we are close to some disk limit and keep hitting it with small changes?

Makes sense to me. It seems like the default github actions [runners](https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories) are 14 GB, and the `.sif` file is about 11 GB according to:

```
$ ll /ocean/projects/deb200005p/stokowsk/containers/dockerst/2024-07-31_15-50-04.sif
```

**Comment by mylesstokowski** [comment]
Thanks for merging @mstrimas

**Pull request merged**